src/canvas-extractor/edfi_canvas_extractor/client_facade.py

```diff
@@ -226,7 +226,8 @@ def extract_enrollments(
         local_enrollments: List[Enrollment] = list(
             enrollmentsApi.request_enrollments_for_section(section)
         )
-        if len(list(local_enrollments)) < 1:
+        filtered_enrollments = [enrollment for enrollment in local_enrollments if enrollment.enrollment_state == "active"]
+        if len(list(filtered_enrollments)) < 1:
             logger.info(
                 "Skipping enrollments for section id %s - No data returned by API",
```

Comment:
Is this comment meaningful? Maybe useful to have _something_, but I don't think this message is very good anymore, especially in light of the adjustment made on line 229.

We still write out a Section Association file, so we're not really "skipping enrollments".

```suggestion
                "There are no no active section associations for section id %s.",
```